README.md

```diff
@@ -1,10 +1,14 @@
-# GOV.UK Chat Evaluation (Prototype)
+# GOV.UK Chat Evaluation
 
-A prototype to explore standardising the approach we use to evaluation on [GOV.UK Chat](https://github.com/alphagov/govuk-chat).
+A CLI tool to run evaluation tasks on [GOV.UK Chat](https://github.com/alphagov/govuk-chat). Tasks typically take a dataset of user input and expected output, they then generate actual user output from GOV.UK Chat, finally the actual output is compared with expected output to produce results.
```

Comment:
That's a very helpful summary section. 

One small suggestion: it might be worth clarifying that each evaluation task is designed to test the performance of a specific component of the chat system. That context helps frame how and why tasks are structured the way they are.

Something like:
```
Each evaluation task is designed to test the performance of a specific component of the chat system, allowing for targeted diagnostics and improvement.
```